code/game/objects/items/tools/misc_tools.dm

```diff
@@ -275,6 +275,25 @@
 	desc = "It's an invisible pen marker."
 	pen_colour = "white"
 
+/obj/item/tool/pen/fountain
+	desc = "A luxurious fountain pen, embossed with gold accents. Its intricate mechanics allow the user to switch between various ink colors with a simple twist."
+	name = "fountain pen"
+	icon_state = "fountain_pen"
+	item_state = "fountain_pen"
+	matter = list("metal" = 20, "gold" = 10)
+	var/list/colour_list = list("red", "blue", "green", "yellow", "purple", "pink", "brown", "black", "orange") // Can add more colors as required
+	var/current_colour_index = 1
```

Comment:
American

Reply:
No. This doesn't look correct. I can change it but please confirm after reviewing the following:
In ss13 we use Color and Colour differently.
Color is for hex values and Colour is for colors written out in text form to be used with Colour matrices.

Reply:
> No. This doesn't look correct. I can change it but please confirm after reviewing the following: In ss13 we use Color and Colour differently. Color is for hex values and Colour is for colors written out in text form to be used with Colour matrices.

https://github.com/cmss13-devs/cmss13/pull/2609

Reply:
> #2609

wtf why would we do this Colour is usually standardized for Colour matrices
past Unga PR moment of people thinking it has to do with UK vs USA spelling

Reply:
> #2609

Also this
![image](https://github.com/cmss13-devs/cmss13/assets/24533979/94845ce6-984c-42db-abe8-a440d9fe71a0)